conan/tools/files/__init__.py

```diff
@@ -1,5 +1,7 @@
 from conan.tools.files.files import load, save, mkdir, ftp_download, download, get, rename, \
-    load_toolchain_args, save_toolchain_args, chdir
+    load_toolchain_args, save_toolchain_args, chdir, unzip, replace_in_file, collect_libs, \
```

Comment:
I'd say lets keep ``load_toolchain_args`` and ``save_toolchain_args`` a implementation detail of the current integrations. Users can save any file they want and recover it easily in ``build()``, no need to standardize.

I am considering exploring the idea of moving to a json compatible with CMakePresets.json, and getting the information from there, from recent feedback it seems doable and it will be much better integrated with IDEs and other stuff

Reply:
Sure, I just kept them because they were already there.